### ISSUE TITLE
Error no kernel image is available for execution on the device

### ISSUE THREAD

**Issue by foxmooner** [Rubiksman78/RenAI-Chat#6]
```
Requirement already satisfied: numpy==1.23.0 in d:\renai-chat-v1.0\libs\pythonlib\lib\site-packages (1.23.0)
INFO:chatbot.model:Loading tokenizer for chatbot_models/pygmalion-2.7b
INFO:chatbot.model:Loading the chatbot_models/pygmalion-2.7b model

===================================BUG REPORT===================================
Welcome to bitsandbytes. For bug reports, please submit your error trace to: https://github.com/TimDettmers/bitsandbytes/issues
================================================================================
Error no kernel image is available for execution on the device at line 479 in file D:\ai\tool\bitsandbytes\csrc\ops.cu
```

**Comment by Rubiksman78** [comment]
Do you have a GPU on your PC and if yes did you do the setup for NVidia,CUDA and cudNN ?

**Comment by foxmooner** [comment]
yeah, my card is GTX 1070 8Gb , NVidia,CUDA and cudNN,All are set up, passed the test, and the versions are aligned to 11.7

**Comment by Rubiksman78** [comment]
Did you use the built-in version from the drive or did you install it yourself with your own python?

**Comment by foxmooner** [comment]
I installed python 3.8.9 in the drive, but when I found that many errors came from the built-in python in the game, I set a virtual environment in the root directory of the game and reinstalled the required modules. Maybe I can try to delete python in the system and download a new version of the game again?

**Comment by Rubiksman78** [comment]
If you made your own installation of python please follow the `Python installation` section on the README or use the built-in with refering the issues related to it.

**Comment by foxmooner** [comment]
Thank you for your reply.
I uninstalled python from the system and re copied the game. 
I only used the configuration in the game. 
Everything was done according to your tutorial, but in the end, the previous error was still displayed. 
The pygmalion model said：
Error no kernel image is available for execution on the device at line 479 in file D:\ai\tool\bitsandbytes\csrc\ops.cu
the rwkv-4-pile-3b model said:
OSError: chatbot_models/rwkv-4-pile-3b does not appear to have a file named config.json. Checkout 'https://huggingface.co/chatbot_models/rwkv-4-pile-3b/None' for available files.
I feel very tired. I don't want to listen to Monica's voice or talk to her by typing before someone makes a real integration package. I gave up. Thank you for doing this. Maybe I can wait for a new version.
